scrapers/ariba/rfp_scraper.py

```diff
@@ -0,0 +1,155 @@
+import datetime as dt
+from pathlib import Path
+from selenium.webdriver.chrome.service import Service as ChromeService
+from selenium.webdriver.common.by import By
+from selenium.webdriver.remote.errorhandler import NoSuchElementException
+from ariba_driver import Ariba
+from time import sleep
+from webdriver_manager.chrome import ChromeDriverManager
+from filemanage import extract_zip_and_move_html, move_pdfs, parse_html, delete_duplicates
+
+# System default download directory
+DOWNLOAD_DIRECTORY = Path.home() / 'Downloads'
```

Comment:
Probably better if we download to a relative `downloads` directory which can be added to `.gitignore`, will be easier to have everything in one place vs. navigating to home/Downloads

Reply:
hmmm there should be an option to add browser options when creating the webdriver to specify the downloads directory. It shouldn't be on the Ariba's end. I can try that out when i get a chance to run the scripts.

Reply:
Seems like we can do something like

`from selenium.webdriver.chrome.options import Options

# Create an instance of Options
chrome_options = Options()

# Set the download directory
download_directory = "/path/to/your/desired/download/directory"

# Add an argument to set the default download directory
chrome_options.add_experimental_option(
    "prefs",
    {
        "download.default_directory": download_directory,
        "download.prompt_for_download": False,
        "download.directory_upgrade": True,
        "safebrowsing.enabled": True,
    },
)

# Create a WebDriver instance with the specified options
driver = webdriver.Chrome(options=chrome_options)`

Reply:
Yup i've updated the python code to add the chrome option, i can verify it's downloading to the `REPO_DIRECOTRY/downloads` now. I've also added a line to created the `data` and `downloads` directory if it doesn't exist. please try it out and let me know

Reply:
Tested and working on Ubuntu 23.04, and MacOS 13.3. I had to make two changes - the download parameter has to be passed if we re-make the driver after an error, and I noticed the login behaviour was slightly different on MacOS so I added a couple of lines to accept that.

Reply:
LGTM